fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchAllTest.java

```diff
@@ -390,7 +407,10 @@ public void testCreatePatientAndObservationWithUniqueTag() throws Exception {
                         .build();
 
         Entity<Patient> entity = Entity.entity(patient, FHIRMediaType.APPLICATION_FHIR_JSON);
-        Response response = target.path("Patient").request().post(entity, Response.class);
+        Response response = target.path("Patient").request()
```

Comment:
Should the test be renamed to indicate that it is testing a different tenant?

Reply:
maybe event a comment explaining what is happening

Reply:
I wanted to just enable it for tenant specific test, but I don't want to change the default behavior - testing the default ds of the default tenant.    I just want to change the tenant in my local to do some simple performance tests against my test tenants.

Reply:
good point, let me add comments

Reply:
done